src/Features/CSharp/Portable/GoToDefinition/CSharpGoToDefinitionSymbolService.cs

```diff
@@ -64,7 +65,7 @@ protected override ISymbol FindRelatedExplicitlyDeclaredSymbol(ISymbol symbol, C
                             return foundReturnableConstruct.GetFirstToken().Span.Start;
                         }
 
-                        return symbol.Locations.FirstOrNone().SourceSpan.Start;
+                        return symbol.Locations.FirstOrDefault()?.SourceSpan.Start ?? 0;
```

Comment:
>symbol.Locations.FirstOrDefault()?. [](http://example.com/codeflow?start=31&length=35)

why not use TryGetFirstLocation?

Reply:
this is at IDE layer, which doesn't have access to those helpers.